common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -86,6 +86,38 @@
 source, template_path = Loader(engine).load_template_source(path)
 %>${source | n, decode.utf8}</%def>
 
+<%def name="sfeTranslate(locale)">
+    <%
+        from django.conf import settings
+        from django.contrib.staticfiles.storage import staticfiles_storage
+        from django.template import Template, Context
+        from webpack_loader.exceptions import WebpackLoaderBadStatsError
+
+        messages = ""
+        messages_path = "{base}/bundles/studio-frontend/locale_messages/{locale}.json".format(
+            base=settings.STATIC_ROOT,
+            locale=locale
+        )
+        with open(messages_path) as inputfile:
+            messages = inputfile.read()
+
+        localeScriptUrl = staticfiles_storage.url(
+            "bundles/studio-frontend/locale_scripts/{locale}.js".format(locale=locale)
+        )
+
+        return Template("""
+            <script type="text/javascript" src={{ localeScriptUrl }}></script>
```

Comment:
This part is broken ATM

Reply:
"Broken" = webpack is not placing the locale script where I expect it to be, so it is not being inserted at this point in templating.